docs/_docs/installation.md

```diff
@@ -50,6 +82,16 @@ gem install jekyll
 All of Jekyll’s gem dependencies are automatically installed by the above
 command, so you won’t have to worry about them at all.
 
+<div class="note info">
+  <h5>Permissions Issues?</h5>
+  <p>
+    The above command performs a system-wide install of jekyll (on most
+    systems). If you get a permissions error, you can either run the command as root
+    (<code>sudo gem install jekyll</code>) or use
+    <a href="https://rvm.io/gemsets/basics">RVM</a> to manage your Gems.
```

Comment:
Isn’t RVM for managing Ruby versions? I have always used RVM in addition to Bundler. Can we provide any more detail here?

Reply:
is there a simpler method than [the one listed here](https://gorails.com/setup/ubuntu/17.10)?

```
sudo apt-get install libgdbm-dev libncurses5-dev automake libtool bison libffi-dev
gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
curl -sSL https://get.rvm.io | bash -s stable
source ~/.rvm/scripts/rvm
rvm install 2.5.0
rvm use 2.5.0 --default
ruby -v
```

Reply:
The only simpler method I can think of is if RVM is in your system's package manager repositories, I'm afraid.